### PR TITLE
mkvtoolnix: update to 85.0

### LINKED ISSUE #50897: qmake build-helper doesn't support qt6
build-style was updated (see #42354, #49741), but not build-helper.

### PR DESCRIPTION
**Pull request by mvf**
#### Testing the changes
- I tested the changes in this PR: **YES**

Fixes #50897

### CODE REVIEW

**Review by Duncaen — DISMISSED** [review]
conflict

**Comment by mvf** [comment]
resolved

**Comment by mvf** [comment]
@Duncaen Any chance you could review this? `mkvtoolnix` is built with [Rake](https://github.com/ruby/rake) and now requires Qt6, so I had to add a new `qmake6` build helper. Changes from the Qt5 `qmake` build helper (in top-down order in the file):

- `qt.conf`: `examples` now sit under `/usr/lib` (previously `/usr/share`)
- `qt.conf`: New key `HostLibraryExecutables` points to `moc` & friends that were moved from `bin` to `libexec`
- Additional `qmake6` alias to match the one in `/usr/lib/qt6/bin` (Debian even has a `/usr/bin/qmake6`)

Thanks!

File: srcpkgs/mkvtoolnix/template [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
-version=78.0
-revision=3
+version=85.0
+revision=1
 build_style=gnu-configure
-build_helper=qmake
-configure_args="--with-docbook-xsl-root=/usr/share/xsl/docbook --enable-qt
- --disable-update-check"
-hostmakedepends="autoconf docbook-xsl gettext libxslt pkg-config qt5-tools-devel ruby"
+build_helper=qmake6
```
Comment (classabbyamp) [inline srcpkgs/mkvtoolnix/template:6]:
```suggestion
build_helper="qmake6"
```

Comment (mvf) [reply srcpkgs/mkvtoolnix/template]: Thanks for taking a look. Pushed!

**Comment by classabbyamp** [comment]
added a little docs

**Comment by mvf** [comment]
Rebased, verified working with Qt 6.7.2.

**Pull request merged**